app/controllers/medicaid/contact_mailing_address_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Medicaid
+  class ContactMailingAddressController < StandardStepsController
+    include MedicaidFlow
+
+    private
+
+    def skip?
+      mail_sent_to_residential? ||
+        no_reliable_mail_address?
+    end
+
+    def mail_sent_to_residential?
+      current_application&.mail_sent_to_residential?
+    end
+
+    def no_reliable_mail_address?
```

Comment:
These methods seem to be duplicated a lot. I wonder if it makes sense to move into the `StandardStepsController` or `MedicaidFlow`? Not a huge deal, just thought it was worth asking.

Reply:
this method in particular just appears once, as far as I can tell. All of these are pretty simple, but they are suppose to help with reading the logic rather than throwing the `!` and `&` stuff all into one big method. I think for now it will be easier to understand if we keep with a little duplication but I will keep my eye on it